internal/http/services/owncloud/ocdav/trashbin.go

```diff
@@ -370,6 +370,18 @@ func (h *TrashbinHandler) restore(w http.ResponseWriter, r *http.Request, s *svc
 
 	sublog := appctx.GetLogger(ctx).With().Logger()
 
+	overwrite := r.Header.Get("Overwrite")
```

Comment:
I would parametrize `"Overwrite"` for easier refactoring, if we ever have to touch this again.